packages/web/src/components/PopupModal/PopupModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, PropsWithChildren } from 'react';
+import {
+  Button,
+  ButtonProps,
+  HStack,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  useDisclosure,
+} from '@chakra-ui/react';
+import { popupProps } from './utils';
+
+interface PopupModalProps extends PropsWithChildren {
+  openModalText: string;
+  header: string;
+  hideCloseButton?: boolean;
+  openButtonProps?: ButtonProps;
+  onConfirm?: () => Promise<void>;
```

Comment:
Am I crazy or is this unused as well? I don't see a single place where an `onConfirm` is passed as a prop... if it's unused, please delete that as well.